src/controllers/expense.controller.js

```diff
@@ -0,0 +1,159 @@
+'use strict';
+
+const { expenseService } = require('../services/expense.service.js');
+const { userService } = require('../services/user.service.js');
+
+const getAll = async(req, res) => {
+  const params = req.query;
+  const expenses = await expenseService.getAll(params);
+
+  res.send(expenses.map(expenseService.normalizeExpense));
+};
+
+const getOne = async(req, res) => {
+  const expenseId = Number(req.params.expenseId);
+  const isExpenseValid = !isNaN(expenseId);
+
+  if (!isExpenseValid) {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  if (!expenseId) {
```

Comment:
This condition is redundant, as the same check is done in line 17. Remove this block.